core/app/models/spree/payment/processing.rb

```diff
@@ -94,7 +94,8 @@ def gateway_options
           # payment gateways happy.
           #
           # For more information, please see Spree::Payment#set_unique_identifier
-          order_id: gateway_order_id
+          order_id: gateway_order_id,
+          originator: self
```

Comment:
I am going to look in to this more. There some confusing usage between `originator` and `action_originator` throughout the code.